src/aead.rs

```diff
@@ -385,6 +385,7 @@ impl Aad<[u8; 0]> {
 }
 
 /// An AEAD key without a designated role or nonce sequence.
+#[derive(Clone)]
```

Comment:
We cannot allow cloning for `UnboundKey` because it would undo the protections that `OpeningKey` and `SealingKey` provide. Please remove the `#[derive(Clone)]` and instead add a comment to this effect.

We can allow cloning only for `LessSafeKey`. I think that adding  `#[derive(Clone)]` for `LessSafeKey` will solve your problem.